.github/workflows/lint.yaml

```
@@ -0,0 +1,87 @@
+name: "Lint"
+
+on: [push, pull_request]
+
+env:
+    fail-fast: true
+
+jobs:
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga
+              with:
+                  args: --diff --dry-run
+
+    linters:
+        name: Linters
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                allow-failures: [false]
+                operating-system: ['ubuntu-latest']
+                php-version: ['7.4']
+
+        steps:
+            - name: "Checkout code"
+              uses: actions/checkout@v2.3.3
+
+            - name: "Install PHP with extensions"
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  coverage: "none"
+                  extensions: intl
+                  php-version: ${{ matrix.php-version }}
+                  tools: composer:v2
+
+            - name: "Set composer cache directory"
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: "Cache composer"
+              uses: actions/cache@v2.1.2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-${{ matrix.php-version }}-composer-${{ hashFiles('composer.json') }}-allow-failures-${{ matrix.allow-failures }}
+                  restore-keys: ${{ runner.os }}-${{ matrix.php-version }}-composer-
+
+            - name: "Require symfony/flex"
+              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-main
+
+            - name: "Composer update"
+              run: composer update --no-scripts
+
+            - name: Lint YAML files
+              run: |
+                  ./bin/console lint:yaml config --parse-tags
+
+            - name: Lint Twig templates
+              run: |
```

If you want to execute all linters steps, no matter there was an error in a previous linter step, you can declare an id `deps` on the composer update step and use:

```suggestion
              if: always() && steps.deps.outcome == 'success'
              run: |
```

on each steps.
So even if the previous step is failing, the next ones are executed and we get all the infos about what is wrong in the PR.

(https://github.com/ogizanagi/symfony-lint-gha-demo/blob/000fcd5b832d6b35a3359d6b07a279445d9d8091/.github/workflows/lint.yaml#L40-L45)

---

Good idea! Let's do that. Thanks!